src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -117,7 +111,6 @@
     /**
      * To string.
      */
-    @SuppressWarnings("unused")
     private final transient CharSequence text;
```

Comment:
@mkordas Yes, I think the same as you. This class (and the text field too) was created by @yegor256.